frontend/src/scenes/authentication/InviteSignup.tsx

```diff
@@ -151,226 +127,171 @@ function AuthenticatedAcceptInvite({ invite }: { invite: PrevalidatedInvite }):
     const { acceptedInviteLoading, acceptedInvite } = useValues(inviteSignupLogic)
 
     return (
-        <div className="authenticated-invite">
-            <Space className="inner" direction="vertical" align="center">
-                <Row>
-                    <h1 className="page-title">You have been invited to join {invite.organization_name}</h1>
-                </Row>
-                <Row>
-                    <span>
-                        You will accept the invite under your <b>existing PostHog account</b> ({user?.email})
-                    </span>
-                </Row>
+        <BridgePage view={'accept-invite'} message={user?.first_name ? `Hey ${user?.first_name}!` : 'Hello!'}>
+            <div className="space-y-2">
+                <h2>You have been invited to join {invite.organization_name}</h2>
+                <div>
+                    You will accept the invite under your <b>existing PostHog account</b> ({user?.email})
+                </div>
                 {user && (
-                    <Row>
-                        <div className="whoami-mock">
-                            <div className="whoami-inner-container">
-                                <WhoAmI user={user} />
-                            </div>
+                    <div
+                        className="border rounded-lg border-dashed flex items-center gap-2 px-2 py-1"
+                        data-attr="top-navigation-whoami"
+                    >
+                        <ProfilePicture name={user.first_name} email={user.email} />
+                        <div className="">
+                            <div className="font-bold">{user.first_name}</div>
+                            <div>{user.organization?.name}</div>
                         </div>
-                    </Row>
+                    </div>
                 )}
-                <Row>
+                <div>
                     You can change organizations at any time by clicking on the dropdown at the top right corner of the
                     navigation bar.
-                </Row>
+                </div>
                 <div>
                     {!acceptedInvite ? (
                         <>
-                            <Button
+                            <LemonButton
                                 type="primary"
-                                block
+                                status="primary-alt"
+                                center
+                                fullWidth
                                 onClick={() => acceptInvite()}
-                                disabled={acceptedInviteLoading}
+                                loading={acceptedInviteLoading}
                             >
                                 Accept invite
-                            </Button>
+                            </LemonButton>
                             <div className="mt-4">
```

Comment:
Nit: `mt-2` might look a bit better here, this looks a bit off to me:
<img width="375" alt="Screen Shot 2022-08-23 at 16 30 14" src="https://user-images.githubusercontent.com/4550621/186185456-92354704-687f-473b-889f-f66dd6a8df64.png">

Reply:
Agreed. Updated